gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-milestone-1-bin.zip
```

Comment:
whoa, should we migrate to m1?

Reply:
I think this is fine as it's only used for building so it only has to work for us. The change is transparent for users. 

I'm looking forward to `7.0` as it comes with Kotlin 1.4 \o/. Which means we can remove `apiLevel = 1.3` in our code and start using 1.4 for real in our code. 

While we're not using any 1.4 API in our code, the same isn't always true about dependencies and I've been hit by https://github.com/Kotlin/kotlinx.atomicfu/issues/171. It's not clear why this wasn't happening before but overall, if we can adopt newer versions of Gradle faster I think that'll help moving forward.

Reply:
PS: talking about other kinds of m1, does K/N support Apple M1 chips ?!?

Reply:
Hm, interesting question, not sure about this.